app/src/main/java/com/anysoftkeyboard/ime/AnySoftKeyboardSuggestions.java

```diff
@@ -709,16 +713,18 @@ protected void handleSeparator(int primaryCode) {
                 // input-box
                 // into "word "->"word, "
                 ic.deleteSurroundingText(1, 0);
-                ic.commitText(((char) primaryCode) + (newLine ? "" : " "), 1);
+                ic.commitText(new String(new int[] {primaryCode}, 0, 1) + (newLine ? "" : " "), 1);
```

Comment:
I see`new String(new int[] {primaryCode}, 0, 1)` is used. Can you extract that into a method and use that? Say `static String codePointToString(int code)` It will give a lot of context.

Reply:
I need to use that in several files. That's the standard code to cast a codepoint into a string; I couldn't find any other way to create a `CharSequence` from the codepoint.

If you can point me to a place where I can define this method just once for the whole project, I'll do it.